### PR TITLE
JEN-957 add trunk secondary job

### PR DESCRIPTION
**Pull request by hors**
https://ps3.cd.percona.com/view/8.0/job/percona-server-8.0-trunk-secondary/7/

### CODE REVIEW

**Review by laurynas-biveinis — COMMENTED** [review]
What does the cron schedule say?
Do you have a test run?

**Comment by hors** [comment]
> What does the cron schedule say?
Do you want to trigger this job as soon as percona-server-8.0-trunk finishes its work?

> Do you have a test run?
I sent the link in the first comment:
https://github.com/Percona-Lab/ps-build/pull/215#issue-300034214

**Comment by laurynas-biveinis** [comment]
@hors sorry for missing the link, it looks good
Yes, please trigger on main trunk job completion

**Review by laurynas-biveinis — CHANGES_REQUESTED** [review]
Please trigger by trunk job completion

File: jenkins/trunk-secondary.yml [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    properties:
+    - build-discarder:
+        days-to-keep: -1
+        num-to-keep: 10
+        artifact-days-to-keep: -1
+        artifact-num-to-keep: 10
+    triggers:
+    - reverse:
+        jobs: 'percona-server-8.0-trunk'
+        result: 'unstable'
```
Comment (laurynas-biveinis) [inline jenkins/trunk-secondary.yml:21]: What if the result is 'stable'?

Comment (hors) [reply jenkins/trunk-secondary.yml:21]: It means the following  'Trigger even if the build is unstable'.

**Review by laurynas-biveinis — APPROVED** [review]

**Pull request merged**